Formula/snapcast.rb

```diff
@@ -20,16 +20,17 @@ class Snapcast < Formula
   depends_on "libsoxr"
   depends_on "libvorbis"
   depends_on "opus"
-  depends_on "pulseaudio"
+
   uses_from_macos "expat"
 
   on_linux do
     depends_on "alsa-lib"
     depends_on "avahi"
+    depends_on "pulseaudio"
   end
 
   def install
-    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
```

Comment:
Might need to inreplace code: https://github.com/badaix/snapcast/blob/v0.26.0/server/snapserver.cpp#L61

Reply:
Bunch of paths are hardcoded (e.g. `/etc`, `/var/run`, `/var/lib`, `/usr/share/snapserver`). Probably needs upstream support, so will probably drop this for now.